crates/consensus/Cargo.toml

```diff
@@ -64,3 +67,4 @@ serde = [
     "dep:alloy-serde",
     "alloy-eips/serde",
 ]
+serde-bincode-compat = ["serde_with"]
```

Comment:
open to renaming this feature to something more generic than bincode, because it also unlocks messagepack via https://github.com/3Hren/msgpack-rust